File: front/client/src/app/Models/Course/Lesson/QuizQuestion/QuizQuestionUpdatedResponse.ts
import { AnswersUpdateResponse } from "./Anwers/AnswersUpdateResponse";
import { CurrentLessonUpdatedResponse } from "../CurrentLessonUpdatedResponse";

export interface QuizQuestionUpdatedResponse {
    id: string;
    question: string;
    answers: AnswersUpdateResponse;
    currentLessonUpdated: CurrentLessonUpdatedResponse;
}

File: src/Learnify/Learnify.Core/Dto/Course/QuizQuestion/QuizQuestionUpdatedResponse.cs
using Learnify.Core.Dto.Course.LessonDtos;
using Learnify.Core.Dto.Course.QuizQuestion.Answers;

namespace Learnify.Core.Dto.Course.QuizQuestion;

public class QuizQuestionUpdatedResponse
{
    public string Id { get; set; }
    public string Question { get; set; }
    public AnswersUpdateResponse Answers { get; set; }
    public CurrentLessonUpdatedResponse CurrentLessonUpdated { get; set; }

}